src/Wrapper/GenericKMSWrapper.php

```diff
@@ -27,6 +27,7 @@ class GenericKMSWrapper implements CryptoWrapperInterface
     private const CONNECT_TIMEOUT = 10;
     private const CONNECT_RETRIES = 5;
     private const TRANSFER_TIMEOUT = 120;
+    private const MAX_RETRIES = 10;
```

Comment:
Kdyz ale na load balanceru a jinde mame limit minutu, tak ty dve minuty jsou naprd ne? By to melo byt min a nebo konfigurovatelny.

Reply:
Hmm, tak kolik bys tam dal 50s ?

Reply:
Dal bych třeba 15/20. A udělal bych to hlavně s možností konfigurace. Tak aby to tak mohlo být u sync věcí. Může to kryptovat/dekryptovat víc věcí naráz tak 50s by mohl být problém.

V commandech atd ať je to klidně 50s.

Reply:
No trošku jsem se nechal unést